app/components/Podcasts.vue

```diff
@@ -20,6 +20,7 @@
 <script>
 import PodcastService from '../api/PodcastService'
 import htmlToText from 'html-to-text'
+import ActionTypes from '../store/constants'
```

Comment:
Since we are not using all constants from /store/constants, could we just destructure the ones we need instead of importing the whole ActionTypes? This could be applied to all files in this PR, not just Podcasts.